tests/bsim/bluetooth/mesh/src/test_access.c

```diff
@@ -98,6 +103,7 @@ static const struct {
 };
 
 static struct k_sem publish_sem;
+static struct k_sem vnd_rx_sem;
```

Comment:
I think both semaphores should be statically initialized, otherwise it will be undefined behavior if some test use wrong message triggering a sempahore which is not initialized.